docs/jwst/straylight/main.rst

```diff
@@ -68,4 +63,14 @@ where,
 
 The radius of influence :math: 'R' and the exponent :math: 'k' are variables that 
 can be adjusted to the actual problem. The default values for these parameters are
-:math:`R = 50` pixels and :math:`k = 1`.
\ No newline at end of file
+:math:`R = 50` pixels and :math:`k = 1`.
+
+
+The second algorithm uses a 'regions' reference file type that is stored in
+the ASDF extension of the input model. The 'regions' information is a 1032 by 1024
+mask containing the 'slice number' of the pixel or in the case of  non-science
+gap pixels, the value zero. The assign_wcs step creates the 'regions' information
+by reading in the MIRI MRS distortion file. The first extension
```

Comment:
Ah, OK, so it sounds like my assumption was correct. Here's the info that verifies that. I just think the flow would be better if the origin of the info were described first, and then describe its format and how it's used. E.g. something along the lines of "The straylight step uses information from the `regions` reference file that was previously loaded into the meta data of the input image by the assign_wcs step. Hence running the assign_wcs step is a prerequisite to the straylight step. The regions information is a 1032 x 1024 ...."